packages/fxa-settings/src/components/Ready/en.ftl

```diff
@@ -1,8 +1,9 @@
 ## Ready component
 
-ready-confirmation = Your password has been reset
+fxa-reset-password-complete-header = Your password has been reset
```

Comment:
Why are these prefaced with `fxa` out of curiosity?

Reply:
oh because those are the existing IDs for the translations! I thought I should preserve them -- should I change them?

Reply:
`id`, which I think you're referring to (like `<h1 id="whatever">`, isn't used by gettext because gettext's translation IDs are tied to their English string counterpart. We're free to do whatever with these new IDs since the translation won't be preserved (gettext translations live in .po files, there's no overlap).